zulipterminal/model.py

```diff
@@ -1686,7 +1685,7 @@ def _handle_update_message_flags_event(self, event: Event) -> None:
         Handle change to message flags (eg. starred, read)
         """
         assert event["type"] == "update_message_flags"
-        if self.server_feature_level is None or self.server_feature_level < 32:
+        if self.server_feature_level == 0 or self.server_feature_level < 32:
```

Comment:
One of the benefits of converting to use zero instead of None is that we don't need multiple conditions in positions like this, since 0 < 32 too.